be/src/storage/lake/tablet_manager.cpp

```diff
@@ -575,8 +577,12 @@ void* metadata_gc_trigger(void* arg) {
                 auto r = metadata_gc(root, tablet_mgr);
                 LOG_IF(WARNING, !r.ok()) << "Fail to do metadata gc in " << root << ": " << r;
             });
-            LOG_IF(WARNING, !st.ok()) << "Fail to submit task to threadpool: " << st;
+            if (!st.ok()) {
+                LOG(WARNING) << "Fail to submit task to threadpool: " << st;
+                break;
```

Comment:
what would happen if continue to next root?

Reply:
Nothing go wrong but has high possibility to fail again.